sample-app/src/components/DropdownSection.tsx

```diff
@@ -12,7 +12,7 @@ export interface DropdownSectionProps {
   optionIdPrefix: string,
   onFocusChange?: (value: string, focusedOptionId: string) => void,
   onLeaveSectionFocus?: (pastSectionEnd: boolean) => void,
-  onClickOption?: (option: Option, optionIndex: number) => void,
+  onSubmit?: (optionIndex: number, option?: Option) => void,
```

Comment:
Do you think `onSelect` might make more sense within the scope of a `DropdownSection`? I can see a 'submission' making sense for a search bar, but for a dropdown, I think 'selection' makes more sense

Reply:
I wasn't sure if 'select' might be confused with focusing on an option. But maybe since there is an 'onFocusChange', it will be clear

Reply:
Updated